packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -435,6 +437,26 @@ class AppDomain extends Domain {
     });
   }
 
+  Future<OperationResult> callServiceExtension(Map<String, dynamic> args) async {
+    String appId = _getStringArg(args, 'appId', required: true);
+    String methodName = _getStringArg(args, 'methodName');
+    Map<String, dynamic> params = args['params'] ?? <String, dynamic>{};
```

Comment:
I think `params` should be `Map<String, String>`, which is how `dart:developer` API [sees them](https://api.dartlang.org/stable/1.21.1/dart-developer/ServiceExtensionHandler.html).